servicex/servicex.py

```diff
@@ -138,10 +138,11 @@ def protected_list_objects(client: Minio, request_id: str):
 async def get_data_async(selection_query: str, datasets: Union[str, List[str]],
                          servicex_endpoint: str = 'http://localhost:5000/servicex',
                          data_type: str = 'pandas',
-                         image: str = 'sslhep/servicex_xaod_cpp_transformer:v0.2') \
+                         image: str = 'sslhep/servicex_xaod_cpp_transformer:v0.2',
```

Comment:
Is there any value in switching this to the v0.4 tag, just to keep it up to date?

Reply:
This is a bigger question - I don't know the right way to do this in ServiceX. Let's start a separate discussion outside of this PR about how to address that - as it might require changes in ServiceX itself.